usaspending_api/etl/management/commands/load_query_to_delta.py

```diff
@@ -34,19 +38,28 @@
         "custom_schema": "recipient_hash STRING, federal_accounts STRING",
     },
     "award_search": {
-        "model": AwardSearch,
+        "model": AwardSearchView,
         "source_query": award_search_load_sql_string,
         "source_database": None,
         "source_table": None,
         "destination_database": "rpt",
-        "swap_table": "award_search",
+        "swap_table": "vw_award_search",
```

Comment:
Looks like some old code mixed in during a merge. Can you remove these changes to the `award_search` `TABLE_SPEC`?